doc/setting_up_sidewalk_environment/migration_guide.rst

```diff
@@ -0,0 +1,161 @@
+.. _migration_guide:
+
+Migration Guide: Moving from NCS Manifest to Standalone west.yml
```

Comment:
Migration Guide for moving to Sidewalk Add-on ?

Reply:
ok